addon/components/au-time-picker.js

```diff
@@ -1,16 +1,35 @@
 import Component from '@glimmer/component';
 import { action } from '@ember/object';
-import { tracked } from '@glimmer/tracking';
 import { isPresent } from '@ember/utils';
+import { localCopy } from 'tracked-toolbox';
+import { formatTimeDigit } from '../helpers/format-time-digit';
 
 export default class AuTimePickerComponent extends Component {
-  @tracked hourValue = this.args.hours || 12;
-  @tracked minuteValue = this.args.minutes || 0;
-  @tracked secondValue =
-    isPresent(this.args.showSeconds) && !this.args.showSeconds
-      ? 0
-      : this.args.seconds || 0; // ignore 'seconds' argument if 'showSeconds' is false
-  @tracked keyCodes = [8, 9, 13, 33, 34, 37, 39, 46];
+  @localCopy('args.hours') hourValue;
+  @localCopy('args.minutes') minuteValue;
+  @localCopy('args.seconds') secondValue;
+
+  constructor() {
+    super(...arguments);
+    this.hourValue = this.hourValue || 12;
+    this.minuteValue = this.minuteValue || 0;
+    this.secondValue = this.secondValue || 0;
+  }
+
+  get hourValueFormatted() {
+    return formatTimeDigit([this.hourValue]);
+  }
+  get minuteValueFormatted() {
+    return formatTimeDigit([this.minuteValue]);
+  }
+  get secondValueFormatted() {
+    return formatTimeDigit([this.secondValue]);
+  }
+
+  //No-operation setters. Input changes handled by events.
+  set hourValueFormatted(newHourValue) {}
+  set minuteValueFormatted(newMinuteValue) {}
+  set secondValueFormatted(newMinuteValue) {}
```

Comment:
It would then complain that (e.g.) `hourValueFormatted` is a getter-only property, so to satisfy JavaScript, I created setters that don't need to do anything, but to accept some value.

Reply:
Hmm, I see. That's probably because of the 2-way-binding in the AuInput component then? Maybe add a todo so that we don't forget to remove these when we [swap out the Input component in the future](https://github.com/appuniversum/ember-appuniversum/issues/250)?

Reply:
So you mean to just add a `//TODO` with some explanation?

Reply:
Yea, with a link to that issue would be nice. It's strange looking code that should be removed ASAP 😄.